web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -52,6 +55,22 @@ const BookingModalForm = props => {
     }
   };
 
+  const remainingDays = (getTotalDays - getApprovedDays);
+  const pendingDays = getPendingDays;
+  const availableDays = (remainingDays - pendingDays);
+  const showRemainingHolsAlertMessage = () => {
+    return (remainingDays === 0 || bookingDuration > remainingDays) || (availableDays < bookingDuration);     
```

Comment:
Is the `remainingDays === 0` part needed? I think `bookingDuration > remainingDays` will cover this saying it's not possible to book 0 day holidays.